ui/src/views/Clients/ViewClients/index.jsx

```diff
@@ -56,14 +56,14 @@ export default class ViewClients extends PureComponent {
     });
 
     if (search !== searchUri) {
-      this.props.history.push(
+      this.props.history.replace(
```

Comment:
This needs to stay as `push`.

Reply:
No changes needed in `ViewClients.js`.

Reply:
ok sir, thanks for the help... i have put it all the way you have directed me. Thanks sir.

Reply:
Np. You have to push your changes for me to see them.

Reply:
I have pushed them sir.

Reply:
Awesome. The last thing to do is to add a changelog. From the root directory (not the ui directory), run `yarn changelog --silent --issue 1558`, then push the changes.

Reply:
Sir, I have pushed it.. Thanks